parameter-sets/oauth-credentials/parameter-set.json

```diff
@@ -4,7 +4,7 @@
         "description": "",
         "icon": "icon-youtube-play"
     },
-    "defaultDefinableInline": true,
+    "defaultDefinableInline": false,
```

Comment:
What does this do ?

Reply:
It makes the default inline option off for the oauth-credential, which in turn should makes unavailable the "Manually defined" option in the user's preset. It make no sense to define the oauth credentials at the datasets / recipes level so overall it's less confusing for the user.